json-schema-store/src/main/java/uk/ac/ebi/biosamples/jsonschemastore/schema/document/SchemaBlock.java

```diff
@@ -0,0 +1,47 @@
+package uk.ac.ebi.biosamples.jsonschemastore.schema.document;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Getter
+@Setter
+@Document(collection = "SchemaBlock")
+public class SchemaBlock {
+  @Id
```

Comment:
How can we store other metadata thats not directly related with the JSON Schema. For example update/create date of the schema, user/group that this schema belongs to, tags related this schema, ......